fix(styles): fix styles for filter selector

edoardo opened the pull request:
Open the filter dialogs within the app container for inheriting the app's
styles.
Fixes DHIS2-6804.

Requires an updated @dhis2/d2-ui-analytics.

edoardo committed:
fix(styles): fix styles for filter selector

Open the filter dialogs within the app container for inheriting the app's
styles.
Fixes DHIS2-6804.

janhenrikoverland reviewed (commented):
Would it work if we e.g. passed in a component instead to avoid the global id?

edoardo committed:
chore(dep): bumped @dhis2/d2-ui-analytics version

edoardo commented:
Found a problem: the page can be scrolled under the open DimensionsPanel.

edoardo committed:
Revert "fix(styles): fix styles for filter selector"

This reverts commit d4a711ee15d0d81c621eae295fd061cc2781f2d0.

edoardo committed:
fix(style): fix font in dimensions panel

Due to the classic plugin still in use in dashboards app, some CSS with
!important rules is added to <body> which makes it impossible to
override those rules at the app level for the modal components that are
added at the end of <body>.
Attaching the modal to another component within the app's shell has the
side effect of enabling page scrolling when the popover is open, which
is not desirable.
A solution is then to force the font-family CSS rule where it would
otherwise inherit it from <body>.

edoardo commented:
Ended up using a different solution.
The modal is still attached to the `<body>` as normal, so the scrolling issue is not happening.
`font-family` is applied to the Popover component.

edoardo committed:
fix(style): use font definition from theme
